### PR TITLE
Submission checks

### PR DESCRIPTION
**Pull request by phlax**
- convert SubmissionType UN/MUTE_CHECK --> SubmissionField.CHECK
- use old_value/new_value to express un/mute

### CODE REVIEW

**Comment by ta2-1** [comment]
lgtm besides "0" and "1" look a bit confusing.

**Comment by phlax** [comment]
replace with MUTED/UNMUTED constants @ta2-1

File: pootle/apps/pootle_statistics/models.py [hunk 1]
```diff
@@ -22,7 +22,9 @@
 from pootle_store.fields import to_python
 
 
+MUTED = "0"
 SIMILARITY_THRESHOLD = 0.5
+UNMUTED = "1"
```
Comment (unho) [inline pootle/apps/pootle_statistics/models.py]: Nit: I would put them together and not with other constant in between.

Comment (phlax) [inline pootle/apps/pootle_statistics/models.py]: im figuring use alphabetical order for constants

**Comment by unho** [comment]
lgtm

File: pootle/apps/pootle_statistics/models.py (hunk 1 above)
Comment (unho) [inline pootle/apps/pootle_statistics/models.py]: Not blocking on it.

**Pull request merged**